glancesync/glancesync_config.py

```diff
@@ -23,6 +23,7 @@
 # contact with opensource@tid.es
 #
 __author__ = 'chema'
+__version__ = '1.2.0.6'
```

Comment:
Why 1.2.0.6?

I suggest to use the __version__ variable from setup.py in order to resolve any future problem.